FedoraApi.php

```diff
@@ -346,6 +346,38 @@ public function getDatastreamDissemination($pid, $dsid, $as_of_date_time = NULL,
     return $response;
   }
 
+  /**
+   * Get the datastream profile.
+   *
+   * @param String $pid
+   *   Persistent identifier of the digital object.
+   * @param String $dsid
+   *   Datastream identifier.
+   * @param String|null $as_of_date_time
```

Comment:
Please document the format accepted by Fedora here
yyyy-MM-dd or yyyy-MM-ddTHH:mm:ssZ